nomad/local/start_detach.sh

```diff
@@ -49,6 +49,16 @@ ensure_valid_nomad_env() {
     ensure_firecracker_driver_installed
 }
 
+# This sets memory oversubscription in Nomad
+# This setting allows containers to exceed memory limits if there is memory available
+set_nomad_memory_oversubscription() {
+    export NOMAD_ADDR="localhost:4646"
+    echo "Setting memory oversubscription"
+    curl -s $NOMAD_ADDR/v1/operator/scheduler/configuration |
+        jq '.SchedulerConfig | .MemoryOversubscriptionEnabled=true' |
+        curl -X PUT $NOMAD_ADDR/v1/operator/scheduler/configuration -d @-
```

Comment:
i doubt these will pass lint, "${NOMAD_ADDR}" them?

Reply:
I'm pretty surprised that they did pass lint. That said I've cleaned this section up so that it matches our normal patterns